### DISCUSSION TITLE
Metadata Storage Question

### DISCUSSION THREAD

**Discussion by SAS-1** [advplyr/audiobookshelf#3051] [General]
Hi

Currently we have an all or nothing setting to either store metadata with each item or in the metadata/items folder.

I was wondering if we could get a setting per library.

For example I would want to store the metadata in the items folder for the Ebooks as Calibre doesn't like additional files in the books folders but store the metadata with the audiobook items for my audio libraries. 

Also I am working on a PowerShell to manage my metadata, am I right in saying that the JSON needs to be called metadata.json I couldn't store a copy with the audiobooks as say booktitle.json and it would read the details? When I did a quick test this seemed to be the case. 

Thanks

**Comment by advplyr** [comment]
Hello, that is correct it needs to be named metadata.json. There is a feature request open for using a different name.

**Reply by advplyr** [reply]
I went ahead and created a feature request for this https://github.com/advplyr/audiobookshelf/issues/3052. If you have any additional comments you can add on to that.
You're the first to bring that one up.

**Reply by SAS-1** [reply]
Thanks
